### ISSUE TITLE
ros2 tf2 don't support frame id starts with /

### ISSUE THREAD

**Issue by cyn-liu** [RoboSense-LiDAR/rslidar_sdk#91]
We recorded bag using RSM1 lidar and found an error while processing point clouds in ros2: tf2 does not support frame id that begin with /. (We use the default frame id: /rslidar)

**Comment by Timple** [comment]
Agreed, namespacing tfs is deprecated even in ROS1 for a long time. So the prepending slash should simply be removed.
